modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP Workload Identity cluster, you must use the `ccoctl` binary to generate the cloud credentials manually and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+* You have configured an {product-title} cluster by using a manual mode with GCP Workload Identity.
+
+.Procedure
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
```

Comment:
Sorry forgot one more thing, you need add below step before line 19:
```
. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
+
[source,terminal]
----
$ mkdir credentials-request

```

And update
```
Create a `CredentialsRequest` resource YAML file
```
to be:
```
Create a `CredentialsRequest` resource YAML file under `credentials-request` directory
```

Reply:
Addressed